Libraries/Utilities/Dimensions.js

```diff
@@ -17,6 +17,12 @@ var RCTDeviceEventEmitter = require('RCTDeviceEventEmitter');
 
 var invariant = require('fbjs/lib/invariant');
 
+type DimensionsChangedEventName = $Enum<{
```

Comment:
`type DimensionsChangedEventName = 'backPress'`

If other events get added here, you can just add them as

`type DimensionsChangedEventName = 'backPress' | 'otherEvent'`